### PR TITLE
Upgrades Tessa for Rails V6

### PR DESCRIPTION
**Pull request by gburgett**
The following updates should be made to any app that upgrades to v6 of this gem:

* This PR removes all the "Asset" model helpers.  
  - `asset :file` declarations become `has_one_attached :file`.  Example: https://github.com/watermarkchurch/watermarkresources.com/pull/1777/commits/fedf006bbbe12a83dee5ab74bf9c9ff1e2963d58
* This PR removes any legacy wrappers.   Example: https://github.com/watermarkchurch/watermarkresources.com/pull/1777/commits/962def8e31af421562098550ef07ca12b4d35972
  - `private_url` should become `.service_url(expires_in: ...)`
  - `public_url` should become `.service_url`
 
* Form objects should no longer pass the ID, but rather the value of the attachment.   Example: https://github.com/watermarkchurch/watermarkresources.com/pull/1777/commits/962def8e31af421562098550ef07ca12b4d35972
  - Change `{attr}_id` to just `{attr}` inside of the `ATTRIBUTES` declaration in a form model  
  - Ensure you are passing along the full attachment object.  ActiveStorage does not merge this in to the `{record}.attributes` method so you'll need to manually merge it in
  - The form post sends an empty string when removing an attachment, but ActiveStorage expects `nil`.  So ensure you call `.presence` when assigning the attachment field.

* The RackUploadProxy is no longer required, because ActiveStorage in Rails 6 has the DirectUploadsController which does the same thing. The Tessa::Engine is still necessary so as to provide the javascript to the asset pipeline, but the engine does not need to be mounted anymore.

refs https://github.com/watermarkchurch/watermarkresources.com/issues/1750

### CODE REVIEW

File: config/routes.rb
Comment (rileyjhardy) [inline config/routes.rb:1]: Does this file still need to exist?

File: lib/tessa/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Tessa
-  VERSION = "2.0"
+  VERSION = "6.0.0.rc1"
```
Comment (rileyjhardy) [inline lib/tessa/version.rb:2]: Why the jump to v6?

**Review by rileyjhardy — APPROVED** [review]

File: lib/tessa/version.rb (hunk 1 above)
Comment (gburgett) [reply lib/tessa/version.rb:2]: The idea is v6 goes with Rails 6

File: config/routes.rb
Comment (gburgett) [reply config/routes.rb:1]: Probably not but it's some work to delete the file, publish a new gem version, and re-test so I'd prefer to leave it.

**Pull request merged**